src/modules/hide_prime_promotions/index.js

```diff
@@ -0,0 +1,45 @@
+const $ = require('jquery');
+const settings = require('../../settings');
+const watcher = require('../../watcher');
+
+const TWITCH_PRIME_BANNER_SELECTOR =
+    '[data-test-selector="test_selector_prime_tracking_button_wrapper"] [data-a-target="blue-bar"]';
+
+class HidePrimePromotionsModule {
+    constructor() {
+        settings.add({
+            id: 'hideTwitchPrimeBanner',
```

Comment:
we should *not* have more settings for this.. one and done. every new setting added is setting bloat